src/lib/pubkey/mce/code_based_key_gen.cpp

```diff
@@ -10,25 +10,139 @@
  */
 
 #include <botan/internal/code_based_key_gen.h>
-#include <botan/code_based_util.h>
-#include <botan/gf2m_rootfind_dcmp.h>
-#include <botan/internal/binary_matrix.h>
+#include <botan/internal/code_based_util.h>
 #include <botan/loadstor.h>
-#include <botan/polyn_gf2m.h>
 
 namespace Botan {
 
 namespace {
 
+#define BITS_PER_U32 (8 * sizeof (u32bit))
```

Comment:
What about adding a BITS_PER_BYTE constant in the `/build-data/arch` section which is 8 for all our target platforms? That way we can make clear to a reader that this is no potential for bugs in case a byte is not 8.